grpc/src/main/java/com/linecorp/armeria/server/grpc/InternalReflectionUtils.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.grpc;
+
+import static org.reflections.util.Utils.isEmpty;
+
+import java.lang.reflect.Method;
+import java.util.List;
+
+import org.reflections.ReflectionUtils;
+
+import com.google.common.base.Predicate;
+import com.google.common.base.Predicates;
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Lists;
+
+/**
+ * Utility class which is used by {@link GrpcServiceBuilder} to get all sorted methods
+ * from a bottom service class.
+ */
+class InternalReflectionUtils extends ReflectionUtils {
```

Comment:
- Should we remove `extends`?
```suggestion
final class InternalReflectionUtils {
```
- Add a private constructor?